roles/kubernetes/node/templates/kubelet.j2

```diff
@@ -12,6 +12,13 @@ KUBELET_ADDRESS="--address={{ ip | default("0.0.0.0") }}"
 # KUBELET_PORT="--port=10250"
 # You may leave this blank to use the actual hostname
 KUBELET_HOSTNAME="--hostname-override={{ ansible_hostname }}"
+{% if inventory_hostname in groups['kube-master'] and inventory_hostname not in groups['kube-node'] %}
```

Comment:
this comes as a `{% if standalone_kubelet|bool %}` as well, could you fix to not repeat things twice?